crates/notion-core/src/inventory/mod.rs

```diff
@@ -159,7 +149,14 @@ impl<D: Distro> Collection<D> {
     }
 }
 
-pub trait Resolve<D: Distro> {
+pub trait FetchResolve<D: Distro> {
+    /// Fetch a Distro version matching the specified semantic versioning requirements.
```

Comment:
I don't fully understand why this trait needs the new `fetch` method. Is this to implement the `notion fetch` command? I'm not opposed to it, I'm just not fully understanding the trait.

Reply:
I created this as a way to get rid of the `fetch_node` and `fetch_yarn` methods, and combine them into a single `fetch` function. And I figured that each `Distro` should have `fetch` methods as well as `resolve_*`.

Reply:
OK, makes sense.